stdlib/polylinesimp/polylinesimp.flux

```diff
@@ -0,0 +1,65 @@
+// Package polylinesimp provides methods for polyline simplication which is an efficient way of downsampling curves retaining the moments of variation throughout the path.
+// This class of algorithms enable efficient rendering of graphs and visualizations without having to load the entire corpus of data points into memory.
+// This is done by reducing the number of vertices that donot contribute significantly to the convexity and concavity of the shape.
+//
+// ## Metadata
+// introduced: 0.7.0
+//
+package polylinesimp
+
+
+// rdp (Ramer Douglas Peucker algorithm) is a polyline simplification function that decimates a curve composed of line segments into a downsampled yet visually indistinguishable curve with fewer points.
+//
+// ## Parameters
+// - valColumn: column that corresponds to the Y axis of the given curve. (optional) (default valColumn: _value)
+// - timeColumn: column that corresponds to the X axis of the given curve. (optional) (default timeColumn: _time)
+// - epsilon: user defined maximum tolerance value that determines the amount of compression. (optional) (epsilon should be greater than 0.0)
+// - retention: user defined retention rate, which specifies the percentage of points to be retained after downsampling. (optional) (Retention rate should be between 0.0 and 100.0)
+// - tables: Input data. Default is piped-forward data (`<-`).
```

Comment:
```suggestion
// - valColumn: Column with Y axis values of the given curve. Default is `_value`.
// - timeColumn: Column with X axis values of the given curve. Default is `_time`.
// - epsilon: Maximum tolerance value that determines the amount of compression. Default is ??.
//
//   Epsilon should be greater than or equal to `0.0`.
//
// - retention: Percentage of points to retain after downsampling. Default is ??.
//
//   Retention rate should be between `0.0` and `100.0`.
//
// - tables: Input data. Default is piped-forward data (`<-`).
```

- What is the default epsilon?
- What is the default retention rate?

Reply:
Both these parameters dont have default values, so if both of them are not provided we run the algorithm in a mode where is automatically learns a good estimate of epsilon for the given data.